Quaver.Shared/Screens/Editor/EditorScreen.cs

```diff
@@ -625,7 +625,7 @@ private void PlayHitsounds()
                 {
                     var view = (EditorScreenView) View;
 
-                    if (ConfigManager.EditorEnableHitsounds.Value && !view.LayerCompositor.ScrollContainer.AvailableItems[obj.EditorLayer].Hidden)
```

Comment:
Reason for removing this? Doing so makes hitsounds played in the editor even if the user doesn't want to.

Reply:
Oh, I completely overlooked the fact that it was a different config option for the editor.

Reply:
Since I think keysounds and hitsounds should be separate settings, should there be a separate `PlayObjectKeySounds` method then?

Reply:
Yes